lib/host/inject.js

```diff
@@ -1,11 +1,13 @@
 var injector = require('connect-injector');
 var debug = require('debug')('testee:html-injector');
+var path = require('path');
+var mime = require('mime-types');
 
 // Returns a middleware that injects the SocketIO JavaScript
 // and a link to the clients side adapters into any HTML file
 module.exports = function(configuration) {
   return injector(function(req, res) {
-    var header = res.getHeader('content-type');
+    var header = res.getHeader('content-type') || mime.contentType(path.extname(req.url));
```

Comment:
Do we need `path.extname` or can it extract right out of `req.url`? Also what happens when the URL has query parameters?